chemprop/args.py

```diff
@@ -749,6 +749,8 @@ class SklearnTrainArgs(TrainArgs):
     """Number of bits in morgan fingerprint."""
     num_trees: int = 500
     """Number of random forest trees."""
+    impute_mode: Literal['model', 'median', 'mean', 'linear','frequent'] = None
```

Comment:
The option `model` is not an especially clear name for what is being done. We could replace it with `single_task` or `single_task_prediction` for more clarity.

Reply:
Agree, will change it!